### ISSUE TITLE
pwd command does not work on windows client

### ISSUE THREAD

**Issue by natewatson999** [zeit/hyper#1214]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10
- **Hyper.app version**: 1.0.0
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**:  N/A
- **Relevent information from devtools** _(CMD+SHIFT+I on Mac OS, CTRL+SHIFT+I elsewhere)_:  N/A
- **The issue is reproducible in vanilla Hyper.app**: `Is Vanilla`. 

## Issue

In the Windows version of Hyper, the `pwd` command does not work. When I attempt to use it, the result is `'pwd' is not recognized as an internal or external command, operable program or batch file.`. What is should do is print the working directory.

The Windows equivalent, using `cd` without arguments, works as per the Windows specification. However, since this is not a Windows-specific tool, what works on other OSes in Hyper should work here, so pwd should be supported.

**Comment by klauscfhq** [comment]
Ohaio! 
From your description I am getting that, you are trying to print your current working directory, while using the **Window Command Prompt**, also known as **cmd.exe**.    
If you would like to do that **while** working on **CMD**, you should use the following command: 

`echo %cd%`

The equivalent, of the above Windows command,  **on** **Linux/Unix** Operating Systems, like Ubuntu for exaple, is **pwd** (**P**rint **W**orking **D**irectory).

Also, if you prefer working on a Linux/Unix environment while on Windows, you should check out the Windows Subsystem for Linux!  [Here](https://msdn.microsoft.com/en-gb/commandline/wsl/install_guide) is some helpful insight from Microsoft on how to enable it on your Windows 10 running machine. It is **still under development**, which means that is not as efficient as the "real thing", so keep that in mind! 😀  

After enabling it you can fire up Hyper and run the following command:

`bash.exe`

That's it! You can now enjoy your Linux Bash on Windows!

**Comment by natewatson999** [comment]
This was using Hyper, not cmd; and I know that on Windows, instead of `pwd`, one uses `cd` without parameters. I know there are ways to get Bash usable on Windows.

I wrote this issue because Hyper seems to at least try to be platform-agnostic, and this is a case of differences between platforms. 

Within Hyper, we use `pwd` on non-Windows platforms, and `cd` on Windows platforms. The latter is because of Windows having a different convention. 

However, Hyper is not a Windows-specific program. If Hyper provides a feature in one manner on all of the other platforms, it should at least be available in Hyper in the same way on Windows; even if it's also available in a different way on Windows. 

Basically, here's what we can use right now for printing the working directory in Hyper:

| Command 	| Windows 	| Non-Windows 	|
|---------	|---------	|-------------	|
| pwd 	| error 	| directory 	|
| cd without parameters 	| directory 	| error 	|

What I am saying is that ideally, it would be this instead when using Hyper:

| Command 	| Windows 	| Non-Windows 	|
|---------	|---------	|-------------	|
| pwd 	| directory 	| directory 	|
| cd without parameters 	| directory 	| error 	|

**Comment by parro-it** [comment]
You measunderstood what hyper is: a Terminal emulator. That is, a gui program that let you run text based programs in a Windows.
The commands you are talking about are exactly that: text based programs that run inside hyper.
Hyper doesn't implement any of them, and it shouldn't.

**Comment by kleinfreund** [comment]
TL;DR: Hyper does not implement `pwd`, `cd`, etc. across platforms.

**Comment by esuus** [comment]
Hmm this is really unfortunate, I'm on windows, I've never used cmd.exe (for obvious reasons), and I've never used `cd` to print my current working directory. I just learned from you now about 'cd' without parameters. So you're telling me in order to use hyper (I just started), I need to learn cmd.exe-specific commands that aren't applicable anywhere else, not even in the Terminal of Visual Studio Code or Powershell.exe?
